### PR TITLE
Added status on outside of task card, Created a dev feature component

### PR DESCRIPTION
**Pull request by skv93-coder**
### Developer Name: @skv93-coder 

Issue Ticket Number:- https://github.com/Real-Dev-Squad/website-status/issues/936

### Description: 
Show the task status on task card.

### Anthing you would like to inform the reviewer about:
No
### Backend changes
No

### Frontend Changes
Yes

### Is Under Feature Flag 
Yes

### Database changes
No

### Breaking changes (If your feature is breaking/missing something please mention pending tickets)
No

Deployment notes
### Any special instructions for the deployment
No
### Tested in staging

No

### Dev Tested:
 Yes

### Images/video of the change:
image

image

image

### CODE REVIEW

File: __tests__/Unit/Components/DevFeature/DevFeature.test.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    it('Should not render anything when there is no dev flag', () => {
+        renderWithRouter(
+            <DevFeature>
+                <p>Children</p>
+            </DevFeature>
+        );
+
+        expect(screen.queryByText('Children')).not.toBeInTheDocument();
+    });
+    it('Should not render anything when dev=true', () => {
```
Comment (ardourApeX) [inline __tests__/Unit/Components/DevFeature/DevFeature.test.tsx:16]: The test case title and the expected result do not comply with each other.

File: src/constants/constants.ts [hunk 2]
```diff
@@ -11,3 +11,16 @@ export const STANDUP_ALREADY_SUBMITTED =
 export const DATEFORMAT = 'MMMM DD, YYYY ';
 export const STANDUPTIME = 6;
 export const TASK_RESULT_SIZE = 5;
+
+export const TASK_STATUS_MAPING: any = {
```
Comment (ardourApeX) [inline src/constants/constants.ts:15]: If possible you can add type here as well.

**Comment by ardourApeX** [comment]
Please add proper PR details. For reference, you can refer to [this](https://discord.com/channels/673083527624916993/729399523268624405/1162449381698588692).

**Review by ardourApeX — CHANGES_REQUESTED** [review]

File: __tests__/Unit/Components/DevFeature/DevFeature.test.tsx (hunk 1 above)
Comment (skv93-coder) [reply __tests__/Unit/Components/DevFeature/DevFeature.test.tsx]: Corrected

**Review by ardourApeX — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**